content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,140 @@
+weight: 70
+keywords: [kubernetes,cni]
+---
+
+The Istio CNI plugin implements the networking details for injecting the Istio sidecar proxy between application pods
+and the container networking infrastructure.  Starting with Istio 1.1, Using the Istio CNI plugin removes the
+requirement for the privileged, `NET_ADMIN` container in the Istio users' application pods.  The
+[Istio CNI plugin](https://github.com/istio/cni) replaces the functionality provided by the `istio-init`
+container injected into application pods' `initContainers` list.
+
+## Prerequisites
+
+1.  Kubernetes installed with the container runtime supporting CNI and `kubelets` configured
+    with the main CNI plugin enabled--`network-plugin=cni`.
+    *  This is enabled for the Kubernetes installation for IBM Cloud IKS, Azure AKS, and AWS EKS clusters.
+    *  For Google Cloud GKE clusters, the
+       [network-policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) feature
+       must be enabled to have Kubernetes configured with `network-plugin=cni`.
+
+1.  Kubernetes installed with the [ServiceAccount admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#serviceaccount) enabled.
+    *  The Kubernetes documentation highly recommends this for all Kubernetes installations
+       where `ServiceAccounts` are utilized.
+
+## Installation steps
+
+The installation of the Istio CNI plugin is integrated with the [Istio Installation with Helm](/docs/setup/kubernetes/helm-install/) procedure via the addition of the setting `--set istio_cni.enabled=true`.
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+  --set istio_cni.enabled=true > $HOME/istio.yaml
+{{< /text >}}
+
+Refer to the full set of `istio-cni` Helm parameters in the chart's [`values.yaml`](https://github.com/istio/cni/blob/master/deployments/kubernetes/install/helm/istio-cni/values.yaml).
+
+For most Kubernetes environments the `istio-cni` chart's defaults will configure the Istio CNI plugin in a
+manner compatible with the Kubernetes installation.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for Kubernetes environment specific procedures.
+
+The following example creates a Istio manifest with the Istio CNI plugin enabled and overrides the default
+configuration of the `istio-cni` Helm chart's `logLevel` and `excludeNamespaces` parameters:
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+  --set istio_cni.enabled=true \
+  --set istio-cni.logLevel=info \
+  --set istio-cni.excludeNamespaces={"istio-system,foo_ns,bar_ns"} > $HOME/istio.yaml
+{{< /text >}}
+
+Note the above `istio-cni.excludeNamespaces` format renders as a json list under the resulting `cni_network_config`
+value `exclude_namespaces`.  The Istio CNI plugin will ignore all pods in those Kubernetes namespaces.
+
+### Helm chart parameters
```

Comment:
There is a lot of repetition on this page from what's in the CNI readme. I wonder if we shouldn't reduce some of this content and just provide a link

Reply:
Yeah, but I hopefully kept it to the details required for Istio administrators here in this doc.  IMO, it makes sense to dup it for now since istio.io is the mechanism for publishing per release docs and separately keeping the current (potentially moving) details within the version specific istio/cni repo could help with istio/cni branch specific PoCs for folks doing bleeding edge stuff.